packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -325,5 +325,6 @@ export default {
         );
     },
     isLintable: true,
+    // @ts-expect-error: Type 'UserInput' is not assignable to type 'PerseusCSProgramUserInput'.
```

Comment:
This is a bummer but I think we saw this coming. I think we need to be using generics here instead of unions.